### PR TITLE
Add cypress tests for platform to pipeline

### PR DESCRIPTION
**Pull request by Kaian**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [ ] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Review by ironArt3mis — CHANGES_REQUESTED** [review]
Functional review required

File: web/portal/platform/cypress/pacts/platform-provider-login-platform-consumer-login.json [hunk 1]
```diff
@@ -1 +1,85 @@
...
+      "providerState": "",
+      "request": {
+        "method": "POST",
+        "path": "/api/platform/admin_login",
+        "headers": {
+          "accept": "application/json, text/plain, */*",
+          "content-length": "38",
+          "content-type": "application/x-www-form-urlencoded"
+        },
+        "body": "username=irontec&password=ee4aeVie7AiH",
```
Comment (Kaian) [inline web/portal/platform/cypress/pacts/platform-provider-login-platform-consumer-login.json:53]: Let's use this opportunity to remove this credentials from our tests

File: Jenkinsfile [hunk 2]
```diff
@@ -384,21 +390,43 @@ pipeline {
...
+                                                    sh '/opt/irontec/ivozprovider/web/portal/platform/bin/test-sync-api-spec platform'
+                                                    sh '/opt/irontec/ivozprovider/web/portal/platform/bin/test-pact'
+                                                }
+                                            }
+                                        }
+                                    }
+                                    post {
+                                        success { notifySuccessGithub() }
+                                        failure { notifyFailureGithub() }
+                                        always { archiveArtifacts artifacts: "web/platform/cypress/screenshots/**/*.png", allowEmptyArchive: true }
```
Comment (Kaian) [inline Jenkinsfile:427]: Missing `portal` in path here. My bad. 

    always { archiveArtifacts artifacts: "web/portal/platform/cypress/screenshots/**/*.png", allowEmptyArchive: true }

**Review by R0MANDEV — APPROVED** [review]

File: web/portal/platform/cypress/pacts/platform-provider-login-platform-consumer-login.json (hunk 1 above)
Comment (rbatistadev) [reply web/portal/platform/cypress/pacts/platform-provider-login-platform-consumer-login.json:53]: and what credentials should I use? You mean to modify those credentials, this kind of test are autogenerated between API response and cypress tests, so if we want to remove it, we should remove the cypress fixture as well and we won't be able to perform the login action.

**Review by ironArt3mis — APPROVED** [review]

**Pull request merged**